styles/jisho/catppuccin.user.css

```diff
@@ -0,0 +1,462 @@
+    // ! We use `display: flex;` instead
+    .button-group {
+      display: flex;
+      flex-wrap: wrap;
+
+      &.radius > *,
+      > li {
+        margin: unset;
+      }
+    }
+
+    .f-dropdown {
+      background-color: @mantle;
+      border-color: @overlay0;
+
+      &::before {
+        border-bottom-color: @surface0;
+      }
+
+      li:hover {
+        background-color: @crust;
+      }
+    }
+
+    header.row {
+      background-color: @base;
+    }
+
+    h1.logo a {
+      background-image: if(
+        @lookup = latte,
+        url("//assets.jisho.org/assets/jisho-logo-v4@2x-7330091c079b9dd59601401b052b52e103978221c8fb6f5e22406d871fcc746a.png"),
+        url("//assets.jisho.org/assets/jisho-logo-v4-dark@2x-e676613b426d34187b61928823730a225b52165aaef99f948bd3dc5fc16fa787.png")
+      );
+    }
+
+    nav .links {
+      .color_theme_picker--wrapper:hover {
+        background-color: @mantle;
+      }
+
+      .color_theme_picker--choices {
+        background-color: @mantle;
+
+        li a:hover {
+          background-color: @crust;
+        }
+      }
+    }
+
+    form.search {
+      .input_methods,
+      .input_method_button h4 {
+        color: @subtext0;
+      }
+
+      .input_method_button:hover {
+        background-color: @surface0;
+      }
+
+      .main {
+        background-color: @surface0;
+        box-shadow: none;
+      }
+
+      .inner {
+        background-color: @surface1;
+
+        input {
+          color: @text;
+        }
+      }
+
+      .search_type {
+        border-right-color: @overlay0;
+        color: @subtext0;
+      }
+
+      .submit {
+        background-color: @overlay1;
+        color: @text;
+      }
+
+      .search-form_clear-button {
+        background-color: transparent; // ! Unsure if this needs a background, also extends beyond the input
+        color: @subtext0;
+      }
+
+      .results,
+      #radical_area .results {
+        .result_label {
+          color: @text;
+        }
+
+        .result {
+          background-color: @surface1;
+
+          &:hover {
+            color: @text;
+          }
+        }
+
+        each(range(8), {
+          .g@{value} {
+            color: @blue;
+          }
+        });
+      }
+
+      #radical_area {
+        .radical_table {
+          background-image: repeating-linear-gradient(
+            180deg,
+            @surface1,
+            @surface1 32px,
+            @surface2 32px,
+            @surface2 64px
+          );
+
+          .number {
+            background-color: @text;
+            color: @base;
+          }
+
+          .radical {
+            background-image: none; // ! some radicals are rendered with a background image by default
+            color: @text;
+
+            &.available:hover {
+              background-color: @overlay0;
+            }
+
+            &.selected {
+              border-color: @text;
+              background-color: mix(@base, @yellow, 50%);
```

Comment:
```suggestion
              background-color: fade(@yellow, 50%);
```

Reply:
I mixed with base to improve the contrast:

`mix(@base, @yellow, 50%)`:
![grafik](https://github.com/user-attachments/assets/b51d5be3-ae6a-4700-8cd0-3e4f15c99232)

`fade(@yellow, 50%)`:
![grafik](https://github.com/user-attachments/assets/c5739e6e-59f1-46e9-8064-e5c1d1779555)

May also want to use crust instead.
`mix(@crust, @yellow, 50%)`:
![grafik](https://github.com/user-attachments/assets/72b10f2c-ca0d-4022-a4fc-c1fd2a7c5114)

Reply:
I don't think we wan't a darkened yellow? We want a faint yellow background. This should look fine, lower to 15-30% if you need to.